tutor/src/components/course-listing/listings.cjsx

```diff
@@ -0,0 +1,155 @@
+_ = require 'lodash'
+React = require 'react'
+BS = require 'react-bootstrap'
+{Link} = require 'react-router'
+classnames = require 'classnames'
+
+Router = require '../../helpers/router'
+
+{CourseStore} = require '../../flux/course'
+{CurrentUserStore} = require '../../flux/current-user'
+
+CourseData = require '../course-data-mixin'
+IconAdd = require  '../icons/add'
```

Comment:
Hm, I was planning to to remove `CourseDataMixin` as part of #1410 since that changes the book splash around enough to require a `CourseTitleBanner` component.  After I used that component everywhere, the mixin wasn't needed. Might make sense to leave it now that you're using it here though.